components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,749 @@
+ + \int_{z_k^{bot}}^{z_k^{top}} \frac{\partial}{\partial z} \left(  \rho \varphi w \right) dz
+= \int_{z_k^{bot}}^{z_k^{top}} \left( D^\varphi + Q^\varphi \right) dz
+$$ (z-integration-tracers)
+
+This results in variables that are vertically mass-averaged, as defined in [](def-mass-thickness-average). The subscript $k$ denotes a variable that has been vertically mass-averaged over layer $k$. This system is now discrete in the vertical, but remains continuous in the horizontal and in time.
+
+$$
+\frac{\partial {\bf u}_k}{\partial t}
++ \zeta_k {\bf u}_k^{\perp}
++ f {\bf u}_k^{\perp}
++ \left[ \rho_k  w_k {\bf u}_k \right]^{top} - \left[ \rho_k w_k {\bf u}_k \right]^{bot}
+=
+- \frac{1}{\rho} \nabla_z p_k
+- \nabla_z K_k
++ {\bf D}_k^u + {\bf F}_k^u
+$$ (layered-momentum-1)
+
+$$
+\frac{\partial h_k}{\partial t} + \nabla_z \cdot \left(h_k {\bf u}_k\right) + \left[ \rho_k w_k \right]^{top} - \left[ \rho_k w_k \right]^{bot}= Q^h_k
+$$ (layered-mass-1)
+
+$$
+\frac{\partial h_k \varphi_k}{\partial t} + \nabla_z \cdot \left(h_k {\bf u}_k \varphi_k\right)
++ \left[ \varphi_k \rho_k w_k \right]^{top} - \left[ \varphi_k \rho_k w_k \right]^{bot}
+=  D^\varphi_k + Q^\varphi_k.
+$$ (layered-tracer-1)
+
+
+Equivalently, one could have multiplied  [](z-integration-momentum), [](z-integration-mass), [](z-integration-tracers) by $g$ and expressed the integrals in terms of pressure, with integration limits from $p_k^{top}$ to $p_k^{bot}$.
+Derivations of ocean model equations with pressure as the vertical variable may be found in [de Szoeke and Samelson 2002](https://journals.ametsoc.org/view/journals/phoc/32/7/1520-0485_2002_032_2194_tdbtba_2.0.co_2.xml) and [Losch et al. 2003](https://journals.ametsoc.org/view/journals/phoc/34/1/1520-0485_2004_034_0306_hsacgc_2.0.co_2.xml). 
+
+The vertical advection terms use the fundamental theorem of calculus to integrate a derivative in $z$, resulting in boundary conditions at the layer interfaces. This makes intuitive sense as a mass balance. For example, in the absence of horizontal advection and sources, the mass equation is simply
+
+$$
+\frac{\partial h_k}{\partial t}
+= \left[ \rho_k w_k \right]^{bot} - \left[ \rho_k w_k \right]^{top}.
+$$ (layered-mass-vert-adv)
+
+This states that the change in mass in the layer is the incoming mass from below minus the outgoing mass above, since the vertical velocity $w$ is positive upwards. The vertical advection of momentum and tracers have a similar interpretation.
+
+### Questions
+
+1. How do integrals in  [](#z-integration-momentum) to [](#z-integration-tracers) turn into the layered form [](layered-momentum-1) to [](layered-tracer-1) when there are products in the terms? For example, $\zeta_k {\bf u}^\perp_k$, $\nabla K$ and $\rho \varphi {\bf u}$.
+1. Using the definition of the layer average, the momentum equation [](layered-momentum-1) should also be thickness-weighted. Perhaps that goes away with conservation of mass?
```

Comment:
I think this deserves more discussion.  I found that the vertically integrated (rather than vertically averaged) barotropic momentum equation was the more sensible one.  Perhaps the same is true of the layered momentum equation.

Reply:
Yes, I need to think about that more. It appears that mass and tracers are vertically mass-averaged (i.e. with factor of $\rho$) and momentum is simply averaged in the vertical without $\rho$. I'm stuck on what happens to the $\Delta z$ in the momentum equation after integrating. Perhaps it just divides out.

Reply:
> It appears that mass and tracers are vertically mass-averaged (i.e. with factor of ρ) and momentum is simply averaged in the vertical without ρ.

While I see what you mean, this doesn't sit well with me.  I think we need to search around more on what others do and figure this out before we consider this settled.

Reply:
The $\Delta z$ doesn't divide out for me. The $\left[ w_k {\bf u}_k \right]^{top} - \left[  w_k {\bf u}_k \right]^{bot}$ term has units of $m^2/s^2$. It looks like you could be missing a factor of layer thickness in the other terms.

Reply:
It could make more sense to present Equation 35 (the pressure gradient components) and then use those to integrate from $z^{bot}$ to $z^{top}$ for Equation 26.